src/main/java/com/google/sps/data/Response.java

```diff
@@ -55,6 +55,14 @@ public String toString() {
         return (this.results).stream().map(n -> n.toString()).collect(Collectors.joining(","));
     }
 
+    public void setPick(Restaurant restaurant){
+        this.pick = restaurant;
+    }
+
+    public void setStatus(String status){
+        this.status = status;
+    }
+
```

Comment:
if you have setter and getter for a "private" attribute, it might as well be public